charts/quickwit/values.yaml

```diff
@@ -424,10 +447,16 @@ prometheusRule:
   #      severity: warning
 
 service:
+  # Deprecated, please use <name>.service.type
   type: ClusterIP
```

Comment:
I agree we can probably get rid of this in favor of per service type config. Maybe we can have a breaking change and remove this entirely.

Reply:
I was trying to avoid breaking changes but yes we could!

Reply:
Just chiming in regarding breaking stuff. We can break things if:
- there's an excellent reason to do so
- we bump the version from 0.5.x to 0.6.x
- the release notes make it very clear what the breaking changes are and provide an easy migration path

Reply:
Of course :smile: 

> there's an excellent reason to do so

I'll let you decide on that! We can support both (in theory indefinitely?), or indeed do a breaking. I suppose avoiding breaking is still the best as long as it doesn't get too complicated to handle it in the templates

Reply:
Ok, it's definitively not important enough to go for a breaking change. Let's keep it with a comment saying:
```
# Service type configuration default for all Quickwit services
```